src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -777,8 +786,9 @@ class HookUtils(object):
         Handler for execjob_launch events.
         """
         pbs.logmsg(pbs.EVENT_DEBUG4, "%s: Method called" % caller_name())
-        # Add the parent process id to the appropriate cgroups.
-        cgroup.add_pids(os.getppid(), jobutil.job.id)
```

Comment:
This line disappeared on the new code. Was this intended? Are we fixing another bug?

Reply:
No it was by accident. Fixed.